doc/source/user_guide/io.rst

```diff
@@ -998,6 +998,7 @@ pass  ``format='mixed'``
 
 .. ipython:: python
 
+   import io
    data = io.StringIO("date\n12 Jan 2000\n2000-01-13\n")
```

Comment:
```suggestion
   data = StringIO("date\n12 Jan 2000\n2000-01-13\n")
```

It seems that all other examples use `StringIO(..)` instead of `io.StringIO(..)`, the the `StringIO` object itself was already imported. So this should fix the error as well.

Reply:
There is one other usage of `io.StringIO` a few lines below that would need to be updated as well (but I can't comment on that line)

Reply:
Yes, removed the import and it fixed the error